src/lipalightninglib.udl

```diff
@@ -65,6 +65,9 @@ interface LightningNode {
     [Throws=DecodeInvoiceError]
     InvoiceDetails decode_invoice(string invoice);
 
+    // Get the max fee strategy that will be employed for paying a given amount in sats
+    MaxFeeStrategy get_payment_max_fee_strategy(u64 amount_sat);
```

Comment:
I thought about doing that, but decided against it because `InvoiceDetails` is not only used for paying an invoice, but is also returned when we create an invoice. I don't think it makes sense to create an invoice and get information about how much it would cost to route such payment.

Reply:
Sure, working on it :)

Reply:
After thinking a bit more, I still think we should use the word `strategy` or something similar, not `fee`. Please check other comments for the reasoning.